cli-output/src/display.rs

```diff
@@ -125,6 +125,37 @@ pub fn println_signers(
     println!();
 }
 
+fn format_account_mode(message: &Message, index: usize, fee_payer_index: Option<usize>) -> String {
+    format!(
+        "{}r{}{}{}{}",
```

Comment:
@t-nelson @CriesofCarrots any suggestions for alternative initials or ordering is very welcome. :)

Reply:
This is really cool.  Feedback after I first intentionally just tried to figure this out by the examples
1. I found it too busy to repeat the flags for each instruction, and my eye kept trying to check the instruction and transaction flags for equality.   How about only showing flags at the transaction level?
2. I figured out "S" quickly.  I couldn't figure out "F" for a long time, and "d" totally escaped me (I brain wanted to say "directory")

Reply:
I ran into the same problem with `d` :slightly_smiling_face: 

I think this could be resolved by adding an [`after_help()`](https://docs.rs/clap/2.33.3/clap/struct.App.html#method.after_help) message with a legend

Reply:
+1 for only putting this information in the tx-level account list

Reply:
tx-level is more precise, too. It'd let you spot those bugs where two instructions specified different permissions and they were compiled into the least restrictive (i.e. one writable, one readable, compiled to writable).